cobra/flux_analysis/variability.py

```diff
@@ -143,7 +158,7 @@ def _fva_optlang(model, reaction_list, fraction):
         fva_old_objective = prob.Variable(
```

Comment:
do you need `m.optimize()` above instead to save the solution? also, I reverted `m.optimize()` to raise an exception when doing the pfba, was not intended as a final solution there but work-in-progress, with that change I guess you don't need to raise `ValueError` here